crates/swc_ecma_preset_env/src/corejs3/data.rs

```diff
@@ -425,28 +476,33 @@ pub static INSTANCE_PROPERTIES: DataMap<&[&str]> = data_map!(Map {
     getYear: "es.date.get-year",
     groupBy: "esnext.array.group-by",
     groupByToMap: ["esnext.array.group-by-to-map", "es.map", "es.object.to-string"],
+    groupToMap:["esnext.array.group-to-map", "es.map", "es.object.to-string"],
     includes: ["es.array.includes", "es.string.includes"],
+    indexed: INDEXED,
```

Comment:
`.indexed` was removed from the iterators helpers proposal, so it makes no sense to add it.